pocs/mount/__init__.py

```diff
@@ -1 +1,56 @@
+from glob import glob
+
 from pocs.mount.mount import AbstractMount  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import load_module
+from pocs.utils.location import create_location_from_config
+from pocs.utils.logger import get_root_logger
+
+
+def create_mount_from_config(config, mount_info=None, earth_location=None, *args, **kwargs):
+    """ Sets up the mount that will be used by the observatory """
+
+    logger = get_root_logger()
+
+    if 'mount' not in config:
+        logger.info("No mount information in config, cannot create.")
+        return None
+
+    if earth_location is None:
+        logger.debug(f'No location provided, using values from config.')
+        site_details = create_location_from_config(config)
+        earth_location = site_details['earth_location']
+
+    if mount_info is None:
+        logger.debug(f'No mount info provided, using values from config.')
+        mount_info = config.get('mount')
+
+    model = mount_info.get('model')
+
+    driver = mount_info.get('driver')
```

Comment:
Please add a check that driver is valid (especially that it is a non-empty string). For example:

```
if not driver or not isinstance(driver, str):
    raise MountNotFound('Mount info in config is missing a driver name.')
```